test/server/grapevine-api/server_spec.coffee

```diff
@@ -670,13 +676,14 @@ describe 'Grapevine API', ->
                 request 'http://localhost:8000'
                   .post '/admin/v1/events'
                   .set 'x-access-token', @token
-                  .send {events: [{'title': 'blah', 'feedID': 1}]}
+                  .send {events: [{'post': 'blah1', 'feedID': 1}, {'post': 'blah2', 'feedID': 1}]}
                   .end (err, res) =>
                     throw err if err
-                    (res.status).should.be.eql 200
+                    (res.status).should.be.eql 201
                     (res.body.message).should.be.eql 'successfully added events'
```

Comment:
It is possible that messages like this might be internationalized and localized some day, so tests should be flexible enough to withstand that